core/src/main/java/io/grpc/internal/ServerCallImpl.java

```diff
@@ -184,6 +185,12 @@ public void setMessageCompression(boolean enable) {
     stream.setMessageCompression(enable);
   }
 
+  @Override
+  @ExperimentalApi("https://github.com/grpc/grpc-java/issues/11021")
```

Comment:
We don't need ExperimentalApi in anything in internal, but we can clean this up after it goes in.